### PR TITLE
Fix `latest-tag-button` position

### LINKED ISSUE #3493: Wrong position of `latest-tag-button`
Extension version: 20.8.20
Link: Any page with `latest-tag-button` (like https://github.com/sindresorhus/refined-github)
Screenshot: 
image

### PR DESCRIPTION
**Pull request by kidonng**
Fix #3493

Test: https://github.com/sindresorhus/refined-github

### CODE REVIEW

File: source/features/latest-tag-button.tsx [hunk 1]
```diff
@@ -102,7 +97,7 @@ async function init(): Promise<false | void> {
 		</a>
 	);
 
-	breadcrumb.after(link);
+	select('#branch-select-menu')!.parentElement!.after(link);
```
Comment (kidonng) [inline source/features/latest-tag-button.tsx:100]: This element always exists, no need to check

File: source/features/latest-tag-button.tsx [hunk 2]
```diff
@@ -139,7 +134,7 @@ async function init(): Promise<false | void> {
 		link.setAttribute('aria-label', 'Visit the latest release');
 	}
 
-	link.classList.add('tooltipped', 'tooltipped-ne');
+	link.classList.add('tooltipped', 'tooltipped-ne', 'rgh-latest-tag-button');
```
Comment (kidonng) [inline source/features/latest-tag-button.tsx:137]: This change isn't related to bugfix but rather #3356. With current changes it is really inconvenient to find a succinct selector to customize the button's size, so I think adding a class won't hurt.

File: source/features/latest-tag-button.tsx (hunk 1 above)
Comment (fregante) [reply source/features/latest-tag-button.tsx:100]: Looks right. We used to use an element that sometimes didn't exist; the selector was changed but not the logic: https://github.com/sindresorhus/refined-github/pull/3353/files

**Review by fregante — APPROVED** [review]

File: source/features/latest-tag-button.tsx (hunk 2 above)
Comment (kidonng) [reply source/features/latest-tag-button.tsx:137]: I should have added it in this line as well 😅 https://github.com/sindresorhus/refined-github/pull/3497/files#diff-5664f505bec7f184c9588cc203936104R107

**Pull request merged**